stylecheck/st1020/st1020.go

```diff
@@ -55,6 +58,10 @@ func run(pass *analysis.Pass) (interface{}, error) {
 		if !ast.IsExported(decl.Name.Name) {
 			return
 		}
+		if _, ok := depr.Objects[pass.TypesInfo.ObjectOf(decl.Name)]; ok && strings.HasPrefix(text, "Deprecated:") {
```

Comment:
When would a comment have the right prefix but not be marked as deprecated by `deprecated.Analyzer`?

Reply:
I put that strings.HasPrefix in so that this:

    // wrong comment yo. //@diag (`comment on exported function`)
    //
    // Deprecated: don't use.
    func F6() {}

Still gets flagged, and it only skips if the `Deprecated: ...` comment is the *only* comment.

Reply:
I'm not confused about the `strings.HasPrefix`, but about the use of `depr`. Isn't the `strings.HasPrefix` call enough?

Reply:
Oh right. Yeah, I guess that it rather redundant an pointless; just seemed like the "proper way" to do it. I removed it.